README.md

```diff
@@ -1,8 +1,23 @@
 # Testify — Android Screenshot Testing
 
+> **Note**
+>
+> 🚚 **We have moved!** 🚧
+>
+> Effective August 1, 2022, all development will occur in our new repository: https://github.com/ndtp/android-testify/
+> 
+>
+
+<a href="https://ndtp.github.io/android-testify/docs/migration"><img width="*" src="./7628.png"/></a>
+
+<sub><sup>Move house vector created by pch.vector - [www.freepik.com](https://www.freepik.com/vectors/move-house)</sup></sub>
```

Comment:
Looks nice (less so in dark mode) but since this is Shopify's repo, what's the copyright of this illustration? Maybe safer to just drop it. IANAL.

Reply:
This line is the required attribution for the image from their site https://www.freepik.com/free-vector/moving-new-apartment-concept_9650123.htm#query=packing%20moving&position=1&from_view=keyword

Reply:
Looks fine in Dark mode to me. What part don't you like?